apps/src/code-studio/initApp/project.js

```diff
@@ -69,9 +69,15 @@ var PathPart = {
  */
 var current;
 var currentSourceVersionId;
-// Server time at which the first project version was saved from this browser tab,
-// for logging purposes.
+// String representing server timestamp at which the first project version was
+// saved from this browser tab, to uniquely identify the browser tab for
+// logging purposes.
 var firstSaveTimestamp;
+// The last time in milliseconds when we forced a new source version.
+let lastNewSourceVersionTime = 0;
```

Comment:
Comment might note "last _client_ time" in contrast to the server timestamp, above.